elements/lisk-p2p/src/peer/outbound.ts

```diff
@@ -135,7 +135,7 @@ export class OutboundPeer extends Peer {
 		outboundSocket.on('connect', async () => {
 			this.emit(EVENT_CONNECT_OUTBOUND, this._peerInfo);
 			try {
-				await Promise.all([this.fetchStatus(), this.discoverPeers()]);
+				await Promise.all([this.fetchAndUpdateStatus(), this.discoverPeers()]);
```

Comment:
Instead of calling with `Promise.all` I think it make sense to call the `fetchAndUpdateStatus` first to check compatibility and then ask for the peerList